c/misra/test/rules/RULE-21-13/test.c

```diff
@@ -0,0 +1,39 @@
+#include <ctype.h>
+#include <stdio.h>
+
+void sample() {
+  unsigned char c1 = 'c';
+  int r1 = isalnum(
+      c1); // COMPLIANT: ASCII 99 is within unsigned char range of [0, 255]
+  int r2 = isalnum(EOF); // COMPLIANT: EOF (-1)
+
+  int x3 = 256;
+  int x4 = x3;
+  int c3 = x4;
+  int r3 =
+      isalnum(c3); // NON_COMPLIANT: is outside unsigned char range of [0, 255]
+
+  unsigned char x5 = EOF;
+  unsigned char x6 = x5;
+  int c4 = x6 + 10000;
+  int r4 =
+      isalnum(c4); // NON_COMPLIANT: is outside unsigned char range of [0, 255]
+
+  int c5 = getchar();
+  int r5 = isalnum(c5); // COMPLIANT[FALSE_POSITIVE]: library functions like
```

Comment:
@jeongsoolee09 I think you modelled `getchar`, do you know why this is still a false positive?

Reply:
It's no longer one, I'm deleting the `FALSE_POSITIVE`.